db/migrate/20151435234636_add_container_node_max_container_group.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerNodeMaxContainerGroup < ActiveRecord::Migration
+  def change
+    add_column :container_nodes, :max_container_groups, :string
```

Comment:
@abonas This is how k8s is exposing it in the api, I'm not sure we want to convert on our own